packages/manager/apps/web-office/src/pages/dashboard/users/Users.tsx

```diff
@@ -15,11 +16,11 @@ export default function Users() {
 
   const { data: dataUsers, isLoading: isLoadingUsers } = useOfficeUsers();
   const {
-    data: dataLicenceDetail,
-    isLoading: isLoadingLicenceDetail,
+    data: dataLicenseDetail,
+    isLoading: isLoadinglicenseDetail,
   } = useOfficeLicenseDetail();
 
-  if (isLoadingUsers || isLoadingLicenceDetail) {
+  if (isLoadingUsers || isLoadinglicenseDetail) {
```

Comment:
`isLoadinglicenseDetail` => `isLoadingLicenseDetail` to be consistent with naming of variables